packages/autocomplete-core/src/getAutocompleteSetters.ts

```diff
@@ -45,9 +45,9 @@ export function getAutocompleteSetters<TItem>({
   };
 
   return {
-    setHighlightedIndex,
+    setSelectedItemId,
     setQuery,
-    setSuggestions,
+    setCollections,
```

Comment:
collection is a list of items internally, while it's still called getSuggestions outside (or do I misread it?)

Reply:
Nope, collections remain internal, except if you use the setters. `getSuggestions` is renamed to `getItems`, and the result of `getItems` is the `items` objet in a collection.

```ts
type Collection = { source, items };
```